src/Request/SearchCityByPostCode.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\MeestBundle\Request;
+
+use Answear\MeestBundle\Enum\RequestEnum;
+use Webmozart\Assert\Assert;
+
+class SearchCityByPostCode implements RequestInterface
+{
+    /**
+     * @var string
+     */
+    private $postCode;
+
+    public function __construct(string $postCode)
+    {
+        Assert::length($postCode, 5);
```

Comment:
I wonder if we should check the length before passing it to the API. Or if the check is to stay shall we check that everything is a number?